recipe/meta.yaml

```diff
@@ -58,14 +38,19 @@ test:
     - docker.utils.ports
     - docker.transport.unixconn
   commands:
-    - python -c "import docker; docker.DockerClient.from_env(version='auto')"  # [win]
-    - python -c "import pkg_resources; pkg_resources.require('docker')"  # [win]
+    - pip check
+    - python -c "import docker; docker.DockerClient.from_env(version='auto')"
+    - python -c "import pkg_resources; pkg_resources.require('docker')"
```

Comment:
This was in the recipe, not sure if it is needed.

Reply:
That is a flaky test anyway. I'm ok removing it.

Reply:
Removed :)